sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -569,7 +569,7 @@ object ScalaJSPluginInternal {
           val semantics = scalaJSLinker.value.semantics
           new RhinoJSEnv(semantics, withDOM = scalaJSRequestsDOM.value)
         } else if (scalaJSRequestsDOM.value) {
-          new PhantomJSEnv(jettyClassLoader = scalaJSPhantomJSClassLoader.value)
+          new JSDOMNodeJSEnv
```

Comment:
It is probably too early for this change. We should give time to people to try out `JSDOMNodeJSEnv` opt-in before we change the default.